README.md

```diff
@@ -13,6 +13,26 @@ In the entry for each XSL file please link to the file in github.
 
 ## Global XSLT
 
+### [/src/doi-underscore.xsl](/src/doi-underscore.xsl)
+
+Solves an issue with doi's that have underscores. A couple of examples of this can be found in: 10.1101/2022.11.10.516056.
+
+```
+<ref id="c59">... doi: 10.1162/jocn{\_}a{\_}01569.</mixed-citation></ref>
+```
+
+should, of course, be:
+
+```
+<ref id="c59">... doi: 10.1162/jocn_a_01569.</mixed-citation></ref>
+```
+
+This is an issue with the bioRxiv XML.
+
```

Comment:
I'm not conformable with implementing this change globally. While you're correct and have identified an error - it's an error that it's in the authors original (see here the reference in their PDF here - https://www.biorxiv.org/content/10.1101/2022.11.10.516056v1.full.pdf). bioRxiv have faithfully captured the DOI as provided by the authors. Since this is an error that is derived form the authors, I suggest that we should also reproduce it (i.e. the broken DOI) on EPP.

The characters which are being globally replaced by this xsl are permitted within DOIs, and implementing this globally could lead to perfectly valid DOIs being broken by this transform.

Reply:
Apologies, I believed that `{, }, and \` were not permitted in DOI. I was mistaken. On re-reading, older DOI's may contain other characters. I will undo this change.